ox-chameleon.el

```diff
@@ -103,7 +118,131 @@ When set to nil, the current theme will be used.")
                         (if (and (< lf 0.4) (< (* sf lf) 0.1)) "#000000" fg)))
               (list bg fg)))))
 
-(defun ox-chameleon--generate-text-colourings ()
+(defun ox-chameleon--symbol->weight (weight)
```

Comment:
I feel like `->` in function names instead of `to` etc. is a stylistic thing that should be done everywhere applicable or not at all, and I'd rather this isn't the odd function out.

Reply:
Muscle memory from lsp-mode, easy change :)